packages/node-core/src/indexer/store.service.ts

```diff
@@ -79,7 +80,7 @@ export class StoreService {
   constructor(
     private sequelize: Sequelize,
     private config: NodeConfig,
-    readonly storeModel: IStoreModelService,
+    readonly storeModel: BaseStoreModelService<ICachedModelControl | IModel<any>>,
```

Comment:
Is there a reason this cant be `IStoreModelService`?

Reply:
Some methods are only available in cacheModel, and not necessary for plainModel, such as `resetData` and `flushData`. However, I don’t think it’s appropriate to define them in `IStoreModelService`.